recipes/zope.hookable/LICENSE.txt

```diff
@@ -0,0 +1,44 @@
+Zope Public License (ZPL) Version 2.1
```

Comment:
The license file is packaged in the pypi tarball, so please remove it from here.